pressbooks/pressbooks
# fix: escape translated text & add network homepage link to dashboard

Fix for #3281. This PR also adds a `Visit homepage` link to the network manager dashboard and bumps some dependencies.

To test:
1. visit your user profile and change your language to French
2. confirm that you can see the 'Institution' field and the helper text (which includes an apostrophe)
![Screenshot from 2023-05-19 17-03-46](https://github.com/pressbooks/pressbooks/assets/13485451/fe7e5333-0bc8-4fa1-96d3-7557994131e1)
3. Change your user language back to English
4. sign in to the network as a network manager or super admin.
5. Confirm that you can see a 'Visit network homepage' link in the top section and that this link takes you to the network homepage
![Screenshot from 2023-05-19 17-05-14](https://github.com/pressbooks/pressbooks/assets/13485451/9cfab2b4-070f-4546-872f-b887d8e352a9)
